src/main/java/org/mule/extension/validation/internal/CommonValidationOperations.java

```diff
@@ -251,6 +257,27 @@ public void isTime(String time, @Optional String locale, @Optional String patter
     validateWith(new TimeValidator(time, nullSafeLocale(locale), pattern, context), context);
   }
 
+  /**
+   * Validates that {@code since} did not occur more than {@code time} amount of {@code timeUnit} units
```

Comment:
`did not` -> the NOT part is wrong. The validator is called isElapsed so you are actually expecting the amount of time to have actually happened. If you need the other case, then add another one called isNotElapsed